File: ObjectiveC Demo/Demo/Demo/ViewController.m
//
//  ViewController.m
//  Demo
//
//  Created by Roger Molas on 9/17/15.
//  Copyright © 2015 KLab Cyscorpions. All rights reserved.
//

#import "ViewController.h"
#import "QRReader.h"

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    [QRReader dataFromView:self.view completionHandler:^(QRReaderReadableCodeObject *code) {
        
        NSLog(@"QR Code:< %@ >",code.stringValue);
        
    } errorHandler:^(NSError *error) {
        
        NSLog(@"Error: <%@ : %@>", error.domain, error.localizedDescription);
    }];
}

@end

File: Swift Demo/SwiftDemo/SwiftDemo/ViewController.swift
//
//  ViewController.swift
//  SwiftDemo
//
//  Created by Keith Samson on 21/09/2015.
//  Copyright © 2015 KLab Cyscorpions. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        
        super.viewDidLoad()

        QRReader.dataFromView(view, completionHandler: { (readableCodeObject) -> Void in
            
            print("QR Code: < \(readableCodeObject.stringValue) >")
            
            }) { (error) -> Void in
                
                print("Error: <\(error.domain) : \(error.localizedDescription)>")
        }
    }
}

